scripts/remove-outdated-translations.js

```diff
@@ -0,0 +1,67 @@
+const fs = require("fs");
+const glob = require("glob");
+const path = require("path");
+
+const keyToUpdate = process.argv[2]; // Get the key to update from command line argument
```

Comment:
Can you add something to the documentation to guide users on how to execute this and what parameters to pass?

Reply:
Definitely, just wanted to get your feedback about whether you think this is a feasible approach going forward.

Reply:
I've now integrated a Github action that directly outputs the command you might consider to run to reset the translations on your branch: 

![image](https://github.com/getAlby/lightning-browser-extension/assets/100827540/b1b45f7b-e0ab-410c-9472-a228a3c77201)